assets/stylesheets/pure-menu.css

```diff
@@ -0,0 +1,197 @@
+/*!
+Pure v0.6.0
+Copyright 2014 Yahoo! Inc. All rights reserved.
+Licensed under the BSD License.
+https://github.com/yahoo/pure/blob/master/LICENSE.md
+*/
+/* HORIZONTAL MENU */
+.pure-menu-horizontal {
+  width: 100%;
+  white-space: nowrap;
+}
+
+.pure-menu-horizontal .pure-menu-list {
+  display: inline-block;
+}
+
+/* Initial menus should be inline-block so that they are horizontal */
+.pure-menu-horizontal .pure-menu-item,
+.pure-menu-horizontal .pure-menu-heading,
+.pure-menu-horizontal .pure-menu-separator {
+  display: inline-block;
+  *display: inline;
+  zoom: 1;
+  vertical-align: middle;
+}
+
+.custom-wrapper {
+  color: white;
+  background-color: black;
+  /*margin-bottom: 1em;*/
+  -webkit-font-smoothing: antialiased;
+  height: 56px;
+  /*overflow: hidden;*/
+  position: relative;
+}
+
+.custom-wrapper.open .pure-menu-list {
+  height: 306px;
+}
+
+.custom-wrapper.open .navigation__logo img {
+  display: none;
+}
+
+.custom-toggle {
+  box-sizing: border-box;
+  display: block;
+  height: 56px;
+  display: block;
+  margin-top: 12px;
+  position: absolute;
+  right: 10px;
+  top: 0;
+  width: 34px;
+  z-index: 4;
+}
+
+.custom-toggle .bar {
+  background-color: #777;
+  display: block;
+  width: 20px;
+  height: 2px;
+  border-radius: 100px;
+  position: absolute;
+  top: 18px;
+  right: 7px;
+  -webkit-transition: all 0.5s;
+  -moz-transition: all 0.5s;
+  -ms-transition: all 0.5s;
+  transition: all 0.5s;
+}
+
+.custom-toggle .bar:first-child {
+  -webkit-transform: translateY(-6px);
+  -moz-transform: translateY(-6px);
+  -ms-transform: translateY(-6px);
+  transform: translateY(-6px);
+}
+
+.custom-toggle.x .bar {
+  -webkit-transform: rotate(45deg);
+  -moz-transform: rotate(45deg);
+  -ms-transform: rotate(45deg);
+  transform: rotate(45deg);
+}
+
+.custom-toggle.x .bar:first-child {
+  -webkit-transform: rotate(-45deg);
+  -moz-transform: rotate(-45deg);
+  -ms-transform: rotate(-45deg);
```

Comment:
Almost positive the browser prefixes are unecessary because `cssnext` uses [autoprefixer](https://github.com/postcss/autoprefixer): https://github.com/cssnext/cssnext/blob/master/package.json#L33